src/App.jsx

```diff
@@ -14,33 +16,89 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button type="button" className="button is-info is-light">
-        Sort alphabetically
-      </button>
+const SORT_ALPHABET = 'alphabet';
+const SORT_LENGHT = 'lenght';
 
-      <button type="button" className="button is-success is-light">
-        Sort by length
-      </button>
+const preparedGoods = (sortMethod, isReverse) => {
+  const gooods = [...goodsFromServer];
 
-      <button type="button" className="button is-warning is-light">
-        Reverse
-      </button>
+  gooods.sort((a, b) => {
+    switch (sortMethod) {
+      case SORT_ALPHABET:
+        return a.localeCompare(b);
+      case SORT_LENGHT:
+        return a.length - b.length;
+      default:
+        return 0;
+    }
+  });
 
-      <button type="button" className="button is-danger is-light">
-        Reset
-      </button>
-    </div>
+  if (isReverse) {
+    gooods.reverse();
+  }
+
+  return gooods;
+};
+
+export const App = () => {
+  const [isGoodSort, setGoodSort] = useState('');
```

Comment:
We mostly use `is` prefix for boolean variables, try to check js some naming convention.


https://www.syncfusion.com/blogs/post/top-javascript-naming-convention.aspx